### ISSUE TITLE
Cors issue with Users endpoint

### ISSUE THREAD

**Issue by adamcodes716** [trakt/api-help#174]
I beat my head against a wall for a few hours trying to get the users endpoint to not have cors issues.  I then tried other endpoints and they worked.  Upon further inspection it looks like the other endpoints have access-control-allow-origin in the response header - I am assuming that this is the reason?

Here is an example of the one that has the cors issue:  
https://api.trakt.tv/users/me/watched/movies&extended=full

**Comment by rectifyer** [comment]
Did you set the CORS origin in your Trakt app settings? It uses that to match and allow the method to be called.

**Comment by adamcodes716** [comment]
Is that in the header?  I've seen various posts on what this should be but I am not sure that I have it right.  I assumed that I did because it only fails on this particular endpoint.  All of the other endpoints work fine with this header.

httpOptions = {
    headers: new HttpHeaders({
      'Content-type': 'application/json',
      'trakt-api-key': 'mykey',
      'trakt-api-version': '2'
      // Authorization : 'Bearer
    })

**Comment by rectifyer** [comment]
No, it would be configured on the Trakt website in your app details.

https://trakt.tv/oauth/applications

**Comment by adamcodes716** [comment]
Thank you again for the response.  Yes, I had that field filled in already with https://adamplays.net (which is where the app lives).  Once again, why is this the only endpoint that fails?    The app works fine right now because it points to a different endpoint.  As soon as I switch the URL it fails.

**Comment by rectifyer** [comment]
CORS is implemented differently on some URLs. Some have "public" CORS for things like metadata, while others have more strict checking of the allowed domains you setup in your app. Please post the full console errors so I can see if that gives any clues.

**Comment by adamcodes716** [comment]
Great, thanks again.  I restored the original URL at https://AdamPlays.net - just click on the link in the menu to access the page if you'd like to access the entire transaction.    I don't think there is anything wrong with rest of the code as it worked fine when I used the chrome extension to get around CORS errors.

SEC7120: Origin http://adamplays.net not found in Access-Control-Allow-Origin header.
XMLHttpRequest: Network Error 0x80700013, Could not complete the operation due to error 80700013
